modules/cmdb/services/audit/audit.go

```diff
@@ -144,7 +142,7 @@ func (a *Audit) convertAuditsToExcelList(audits []model.Audit) ([][]string, erro
 		return nil, err
 	}
 	for _, u := range users {
-		userIDNameMap[strconv.FormatUint(u.ID, 10)] = u.Nick
+		userIDNameMap[u.ID] = u.Nick
 	}
 
 	// 把r里的userID替换成userName
```

Comment:
Bad annotation.

Reply:
that's old stuff, leave here, we have a plan to refactor all of codes. until then, all chinese text will be deleted.